src/xform_to_datalog/authorization_logic_datalog_facts.cc

```diff
@@ -23,38 +23,29 @@
 
 #include "absl/strings/str_cat.h"
 #include "absl/strings/str_join.h"
-#include "src/xform_to_datalog/authorization_logic.h"
 #include "src/common/logging/logging.h"
+#include "src/xform_to_datalog/authorization_logic.h"
 
 namespace raksha::xform_to_datalog {
 
 std::optional<AuthorizationLogicDatalogFacts>
-AuthorizationLogicDatalogFacts::create(
-  const std::filesystem::path &program_dir, absl::string_view program) {
+AuthorizationLogicDatalogFacts::create(const std::filesystem::path &program_dir,
+                                       absl::string_view program) {
   const std::filesystem::path result_dir =
-    std::filesystem::temp_directory_path();
+      std::filesystem::temp_directory_path();
 
   // List of relations to not declare in the generated auth logic code
   // because we already have definitions within the Raksha dataflow files for
   // this purpose. We use an vector here and create the composite string to pass
   // into the Rust code with StrJoin to make the code a bit more readable and to
   // prevent typos changing the interpretation of the list.
   const std::vector<absl::string_view> kRelationsToNotDeclare{
-      "says_ownsTag",
-      "says_ownsAccessPath",
-      "says_hasTag",
-      "says_canSay_hasTag",
-      "says_removeTag",
-      "says_canSay_removeTag",
-      "says_may",
-      "says_will",
-      "isAccessPath",
-      "isTag",
-      "isPrincipal"
-  };
+      "says_ownsTag", "says_ownsAccessPath", "says_hasTag",  "says_removeTag",
```

Comment:
The only change in this file was that the _canSay_ relations are removed from this list because now auth logic can generate those with the types that the C++ side is expecting.